evap/static/scss/components/_tables.scss

```diff
@@ -112,12 +112,6 @@
     }
 }
 
-.following-hover-row-disabled + .hover-row:hover {
```

Comment:
I'm sorry for pushing more commits, even though I said I'm done here, but this solution came in my mind when I tried responsive designs for the results pages...

Just let me know when you prefer a new rebase when those !fixups are getting on your nerves